frontend/pages/blog/[id].tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import { useRouter } from 'next/router';
+import {BlogApi} from "../../hooks/blogs"
+import Link from "next/link";
+import styles from "../../styles/Blog.module.css"
+
+
+
+
+const BlogView=()=>{
+    const router = useRouter();
+    const { id } = router.query;
+    const [blogdata, setBlogdata] = useState([]);
+  const {mutate ,isLoading} = BlogApi((res)=>{ 
+    console.log('Mutation successful!', res);
+      // Update the posts state with the updated data from the response
+      setBlogdata(res.data);
+  },(err)=>{
+    console.log('Mutation error:', err);
+  })
+    // console.log("hihi")
+  
+   useEffect(()=>{
+    mutate();
+   },[])
+
+   const blog = blogdata.find((blog) => blog.id ==id);
+   console.log(blog)
+   if (!blog) {
+    return <div>Blog post not found</div>;
+  }
+    
+    const styleObj = {
```

Comment:
border radious jaise cheez ko tailwind use karo and rest ko common CSS file me daldo